### PR TITLE
"ipfs log ls" command

### PR DESCRIPTION
**Pull request by hsanjuan**
#2434 

Summary of changes:
- go-log updated to QmaDNZ4QMdBdku1YZWBysufYyoQt1negQGNav6PLYarbY8
- logging.Uuid() calls removed or fixed, since this function has gone away from go-log
- goimports has re-ordered some imports and added some line breaks
- added ls commands to core/commands/log.go

### CODE REVIEW

**Comment by whyrusleeping** [comment]
Sweet, sorry about dropping the uuid thing without telling you. Could you add it back into the 'thirdparty' logging extension:

```diff
diff --git a/thirdparty/loggables/loggables.go b/thirdparty/loggables/loggables.go
index 17db9ac..dbf3d1f 100644
--- a/thirdparty/loggables/loggables.go
+++ b/thirdparty/loggables/loggables.go
@@ -14,6 +14,8 @@ import (
        logging "gx/ipfs/Qmazh5oNUVsDZTs2g59rq8aYQqwpss8tcUWQzor5sCCEuH/go-log"
 
        peer "gx/ipfs/QmZwZjMVGss5rqYsJVGy18gNbkTJffFyq2x1uJ4e4p3ZAt/go-libp2p-peer"
+
+       uuid "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/satori/go.uuid"
 )
 
 // NetConn returns an eventlog.Metadata with the conn addresses
@@ -31,6 +33,12 @@ func Error(e error) logging.Loggable {
        }
 }
 
+func Uuid(key string) logging.Metadata {
+       return logging.Metadata{
+               key: uuid.NewV4().String(),
+       }
+}
+
 // Dial metadata is metadata for dial events
 func Dial(sys string, lid, rid peer.ID, laddr, raddr ma.Multiaddr) DeferredMap {
        m := DeferredMap{}
```

And then call it from there instead of removing it?

**Comment by hsanjuan** [comment]
Now let's see if the tests don't time out..

File: thirdparty/loggables/loggables.go [hunk 1]
```diff
@@ -9,9 +9,11 @@ package loggables
 import (
 	"net"
 
+	uuid "github.com/satori/go.uuid"
```
Comment (whyrusleeping) [inline thirdparty/loggables/loggables.go]: need to change this to `github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/satori/go.uuid`. Gotta make sure we always vendor our dependencies.

Comment (hsanjuan) [inline thirdparty/loggables/loggables.go]: Sorry for that, my brain needs rest.

**Comment by whyrusleeping** [comment]
Cool, thanks for the PR!

**Pull request merged**